hazelcast/src/main/java/com/hazelcast/config/CachePartitionLostListenerConfig.java

```diff
@@ -74,12 +74,7 @@ public boolean equals(Object o) {
         }
 
         CachePartitionLostListenerConfig that = (CachePartitionLostListenerConfig) o;
-
-        if (className != null ? !className.equals(that.className) : that.className != null) {
```

Comment:
This code was redundant as it's already checked in the `super.equals()` method. I added tests for this, so the removal should be fine. This dead code just hurts the code coverage.